pages/about/adr-process.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: About
+permalink: /about/adr-process
+layout: default
+---
+
+== Github Process
+
+=== Proposed ADRs
+
+If you identify the need for an ADR please {{site.github.repository_url}}/issues/new[open an issue] describing the decision that needs to be taken.
+
+The discussion of the issue should decide whether a new ADR really is needed.
+If so, it should also seek to identify.
+
+* Whether any {{site.url}}/padr/[platform ADRs] apply. 
+* What tags the ADR should have. 
+* What the _summary_ of the new ADR should be. 
+
+=== Drafting an ADR
```

Comment:
I can automate that step easily after.
When approved label is added to issue we can trigger bot that will

close issue
Create PR with title and number from issue.